content/v1.0.0-beta.2/index.md

```diff
@@ -31,7 +31,7 @@ consumers of your library:
   with [`MINOR`](http://semver.org/#summary) in semantic versioning).
 1. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`MAJOR`](http://semver.org/#summary) in semantic versioning). A breaking change can be
   part of commits of any _type_. e.g., a `fix:`, `feat:` & `chore:` types would all be valid, in addition to any other _type_.
-1. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional) (based on the [the Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others. We also recommend `improvement` for commits that improve a current implementation without adding a new feature or fixing a bug. Notice these types are not mandated by the conventional commits specification, and have no implicit effect in semantic versioning (unless they include a BREAKING CHANGE, which is NOT recommended).
```

Comment:
This changes should not be there 😄

Reply:
`[commitlint-config-conventional](https://github.com/marionebl/commitlint/tree/master/%40commitlint/config-conventional)`
is a dead link now, that repo has moved to a new place
I just updated the link.

Reply:
So let's fix it in another PR (or at least change the name of the PR)

Reply:
Oh yeah, you are right, thanks
(never thought that before... 😥)
I'll make a new commit to reverse (manually) these changes and a new PR soon

Reply:
@damianopetrungaro 
I'm thinking should I rework my branch to only output fewer cleaner commits (is that ok for the github review system?)
or close this PR and make a brand new one
or is current commit history for this PR acceptable